tests/phpunit/tests/Story_Post_Type.php

```diff
@@ -289,4 +289,30 @@ public function test_add_caps_to_roles() {
 			$editor->has_cap( $cap );
 		}
 	}
+
+
+	/**
+	 * @covers ::add_caps_to_roles
+	 * @covers \Google\Web_Stories::new_site
+	 */
+	public function test_add_caps_to_roles_multisite() {
+		if ( ! is_multisite() ) {
+			$this->markTestSkipped( 'Test only runs on Multisite' );
+		}
```

Comment:
Can't you just add `@group ms-required`? That does the same.

See https://github.com/WordPress/wordpress-develop/blob/022ebde54d6f06556382fa34f6e8d5fb0ee24d03/tests/phpunit/includes/abstract-testcase.php#L801-L803

Reply:
This was copied from 

https://github.com/google/web-stories-wp/blob/main/tests/phpunit/tests/Updater.php#L175-L177 
https://github.com/google/web-stories-wp/blob/7cf07181c4a39995df3552a4f50b86036c2b43af/tests/phpunit/tests/Uninstall.php#L76-L78

Reply:
Should I change these as well?

Reply:
Then let's fix it everywhere :-)

Reply:
Fixed